src/ui/Alert/Alert.spec.tsx

```diff
@@ -86,15 +86,16 @@ describe('Alert', () => {
 
   describe('Custom Icons', () => {
     const alert = (
-      <Alert variant={AlertOptions.INFO} customIconName="sparkles">
+      <Alert variant={AlertOptions.INFO} customIconName="speakerphone">
```

Comment:
this seems a little random lol

Reply:
TS was failing to build with Vite, sparkles doesn't appear to be a valid icon name.